src/Infrastructure/Memo4me.Persistence/Extensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Memo4me.Persistence;
+
+public static class Extensions
+{
+    public static void AddPersistence(this IServiceCollection services)
+    {
+        services.AddDbContext<AppDbContext>(x =>
+        {
+            x.UseNpgsql("Username=postgres;Password=123;Host=db;Port=5432;Database=MemoDb");
```

Comment:
should be passed as a configuration parameter

Reply:
Read more about configuration https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-8.0